draft-ietf-mls-protocol.md

```diff
@@ -1329,18 +1329,30 @@ struct {
     uint16 length = Length;
     opaque label<7..255> = "mls10 " + Label;
     opaque context<0..2^32-1> = Context;
+    uint8_t count = Count;
+    uint8_t index = Index;
 } HKDFLabel;
 
-Derive-Secret(Secret, Label) =
-    HKDF-Expand-Label(Secret, Label, "", Hash.length)
+Derive-Secret(Secret, Label, Context) =
+    HKDF-Expand-Label(Secret, Label, Context, 0, 1, Hash.length)
+
+Derive-Secrets(Secrets, Count, Label, Context) =
+        HKDF-Expand-Label(Secret0, Label, Context, 0,   n, Hash.length)
+    xor HKDF-Expand-Label(Secret1, Label, Context, 1,   n, Hash.length)
+              ⁝
+    xor HKDF-Expand-Label(Secret1, Label, Context, n-1, n, Hash.length)
```

Comment:
Are you sure you don't mean Secretn here?

Reply:
Thanks! Fixed that.